j2cl/src/main/webapp/assets/sidecar.css

```
@@ -201,6 +201,73 @@ body {
+
+.j2cl-read-blip-meta {
+  color: #4f6480;
+  font-size: 0.82rem;
+  font-weight: 700;
+  margin-bottom: 8px;
+}
+
+.j2cl-read-blip-content {
+  line-height: 1.65;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)